src/modules/python/pbs_hooks/PBS_cray_atom.PY

```diff
@@ -176,6 +177,60 @@ class HookHelper(object):
         log_with_caller(pbs.EVENT_DEBUG4, 'path is %s' % path)
         return path
 
+    def is_it_exclusive(job):
+        """
+        check to see if the job requested exclusive, or if the
+        nodes are marked exclusive.  This needs to be passed
+        to ATOM.
+        """
+        place = job.Resource_List["place"]
```

Comment:
you use str(place) everywhere you use place, so you might as well just str it here.